opencti-platform/opencti-front/src/private/components/analyses/notes/NoteEdition.tsx

```diff
@@ -32,6 +34,23 @@ const NoteEdition = ({ noteId }: { noteId: string }) => {
     });
   };
 
+  // Can't use EditEntityControlledDial because it's too small for some reason.
```

Comment:
could you make EditEntityControlledDial  configurable with a props to overwrite default variant instead ?